esx_service/utils/kvESX.py

```diff
@@ -166,35 +168,45 @@ def delete(volpath):
     return True
 
 
+# Align a given string to the specified block boundary.
+def align_str(kv_str, block):
+   # Align string to the next block boundary.
+   new_len = int((len(kv_str) + block - 1) / block) * block - 1
+   padded_str = '{:<{width}}'.format(kv_str, width=new_len)
```

Comment:
the single line I suggested is returning **exactly** the same string (including newline), so I am not sure what are you talking about